graylog2-server/src/main/java/org/graylog2/rest/resources/system/responses/LookupTableCachePurgingNodeResponse.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.rest.resources.system.responses;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LookupTableCachePurgingNodeResponse {
+    public final String nodeId;
+    public final boolean success;
+    public final String message;
+
+    @JsonCreator
+    private LookupTableCachePurgingNodeResponse(
+            @JsonProperty("success") boolean success,
+            @JsonProperty("nodeId") String nodeId,
```

Comment:
We use `snake_case` instead of `camelCase` for our field names in JSON. So this should be `node_id`.

Reply:
1f1e1810e759d42ac57c9e3956202c653e2c9ba7